src/Evolution/Systems/CurvedScalarWave/PsiSquared.hpp

```diff
@@ -0,0 +1,29 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/CurvedScalarWave/TagsDeclarations.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace CurvedScalarWave::Tags {
+struct PsiSquared : db::SimpleTag {
```

Comment:
did you want to add a short doxygen comment just so these tags show up in the documentation?